contracts/validator-manager/PoSValidatorManager.sol

```diff
@@ -141,6 +141,10 @@ abstract contract PoSValidatorManager is
         {
             revert InvalidStakeMultiplier(maximumStakeMultiplier);
         }
+        // Minimum stake duration should be at least one churn period in order to prevent churn tracker abuse.
+        if (minimumStakeDuration < _getChurnPeriodSeconds()) {
```

Comment:
Does our expected churn period for most deployments align well with our expected minimum stake duration? For example, if minimunStakeDuration is set to the current minimum (2 weeks), is a churn period of 2 weeks or less a reasonable setting?

Reply:
I think a reasonable setting for the churn period is probably on the order of 1-10 hours. Some users might opt for a very low minimum stake duration, so this is to sanity check against that.